src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -67,7 +67,7 @@ export default class DefaultBrowserBehavior implements BrowserBehavior {
   }
 
   requiresUnifiedPlan(): boolean {
-    let shouldEnable = this.isSafari() || this.isFirefox();
+    let shouldEnable = (this.isSafari() && this.isUnifiedPlanSupported()) || this.isFirefox();
```

Comment:
it's safer to check isUnifiedPlanSupported for all browsers and not just for safari.

Reply:
Will add the check there

Reply:
@XHatan I think @kaaaren wrote the Safari Unified Plan checker based on this WebKit article:
https://webkit.org/blog/8672/on-the-road-to-webrtc-1-0-including-vp8/

We might need another Unified Plan checker for Chrome using `addTransceiver` but only if we plan to support lower versions: https://chromestatus.com/feature/4749621767176192